core/src/main/java/discord4j/core/object/entity/ApplicationInfo.java

```diff
@@ -461,6 +464,62 @@ public Flux<ApplicationRoleConnectionMetadata> setRoleConnectionMetadata(List<Ap
             .map(ApplicationRoleConnectionMetadata::new);
     }
 
+    /**
+     * Requests to retrieve the application's emojis.
+     *
+     * @return A {@link Flux} that continually emits guild's {@link ApplicationEmoji emojis}. If an error is received, it is
+     * emitted through the {@code Flux}.
+     */
+    public Flux<ApplicationEmoji> getEmojis() {
+        return gateway.rest()
+            .getEmojiService()
+            .getApplicationEmojis(this.getId().asLong())
+            .flatMapIterable(ApplicationEmojiDataList::items) // the response include a items field with all the emojis
+            .map(emojiData -> new ApplicationEmoji(gateway, emojiData, getId().asLong()));
+    }
+
+    /**
+     * Requests to retrieve the guild emoji as represented by the supplied ID.
+     *
+     * @param id The ID of the guild emoji.
+     * @return A {@link Mono} where, upon successful completion, emits the {@link GuildEmoji} as represented by the
+     * supplied ID. If an error is received, it is emitted through the {@code Mono}.
+     */
+    public Mono<ApplicationEmoji> getApplicationEmojiById(final Snowflake id) {
```

Comment:
Either rename this method "getEmojiById" or just "getEmoji" to keep a similarity between all methods names.

Reply:
i based in Guild where has the same names. https://github.com/Discord4J/Discord4J/blob/3.2.x/core/src/main/java/discord4j/core/object/entity/Guild.java#L533